gulp/tasks/gen-service-worker.js

```diff
@@ -86,13 +85,29 @@ gulp.task('gen-service-worker', () => {
         rootDir + '/fonts/roboto/Roboto-Bold.ttf',
         rootDir + '/images/card_media_player_bg.png',
       ],
-      runtimeCaching: [{
-        urlPattern: /\/static\/translations\//,
+      // Rules are proceeded in order and negative per-domain rules are not supported.
+      runtimeCaching: [
+        { // Cache translations on first access
```

Comment:
Since we don't fingerprint languages, will this mean that new translations won't be fetched?

Reply:
We do fingerprint languages.

Reply:
Oh, perfect!